app/javascript/components/patient/close_contact/CloseContact.js

```diff
@@ -16,12 +16,14 @@ import DateInput from '../../util/DateInput';
 import InfoTooltip from '../../util/InfoTooltip';
 import PhoneInput from '../../util/PhoneInput';
 import reportError from '../../util/ReportError';
+import DeleteDialog from '../../util/DeleteDialog';
 
 class CloseContact extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       showModal: false,
```

Comment:
Can you rename this `showCloseContactModal` to be more clear how this is different than the delete modal?  Same with `toggleModal` -> `toggleCloseContactModal` and `submit` -> `handleCloseContactSubmit`

Also order/group these so all the delete methods are together, and all the close contact methods are together